app.vue和header.vue

Kate605690919 opened the pull request:
header组件在app.vue中引用，不用在每个页面中在引用了；header的样式更改，整理了一下略混乱的css，不过应该还可以写的更简略；html换成了100px，所有计算使用rem。修改了上一个pr中header的props的问题。

605690919 committed:
header，app

605690919 committed:
rem更改

605690919 committed:
props更改，之前报错没看见

Yann001 commented:
有一个小bug，写文章按钮 hover 之后看不见字了，请修改后再发pr，请尽量使用不同分支完成不同事情，发一个pr，然后头像旁边的提示三角我觉得是可以加的，你自己看效果吧

Yann001 commented on src/components/header.vue:
```diff
@@ -0,0 +1,254 @@
+            border-radius: 50%;
+          }
+        }
+        ul {
```
是不是应该用class好一点？

605690919 committed:
header的hover样式更改，其他间距微调

605690919 committed:
文件路径更改

放在assets里的会被webpack打包，static中应该放一些别人写的，不自己改的代码，区别见https://segmentfault.com/q/1010000009842688
